google/cloud/storage/grpc_plugin.h

```diff
@@ -46,19 +57,14 @@ struct [[deprecated(
 /**
  * Create a `google::cloud::storage::Client` object configured to use gRPC.
  *
- * @param opts the configuration parameters for the Client.
- *
- * @warning At present, GCS gRPC is GA with Allowlist. To access this API,
- *   kindly contact the Google Cloud Storage gRPC team at
- *   gcs-grpc-contact@google.com with a list of GCS buckets you would like to
- *   Allowlist. Please note that while the **service** is GA (with Allowlist),
- *   the client library features remain experimental and subject to change
- *   without notice.
- *
- * @par Example
- * @snippet storage_grpc_samples.cc grpc-read-write
+ * @deprecated Please use `google::cloud::storage::MakeGrpcClient`.
  */
-google::cloud::storage::Client DefaultGrpcClient(Options opts = {});
+[[deprecated(
+    "use ::google::cloud::storage::MakeGrpcClient() instead")]] inline google::
```

Comment:
Why the `inline`? Consider dropping it and defining this function in the source file.

Reply:
Done.